smartsheet-csharp-sdk/main/Smartsheet/Api/Models/ScheduleType.cs

```diff
@@ -28,10 +28,25 @@ namespace Smartsheet.Api.Models
     /// </summary>
     public enum ScheduleType
     {
+        /// <summary>
+        /// Shedule only to happen once
```

Comment:
typo in these schedules

Reply:
Fixed in my latest commit please check now. Thanks for the eagle eyes! 🦅

Reply:
doh sorry. thanks!